src/Hud/XpRate/XpRatePlugin.cs

```diff
@@ -108,8 +108,14 @@ public override void Render()
                         if (dif < 0) { bounds.X += dif; bounds.Width -= dif; }
 
                         Graphics.DrawText(timer, Settings.TextSize, new Vector2(bounds.X + 70, position.Y), Settings.TimerTextColor);
-                        Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
-                        Graphics.DrawText(ping, Settings.TextSize, new Vector2(bounds.X + 70, thirdLine.Y), Settings.LatencyTextColor);
+                        if (Settings.ShowFps)
+                        {
+                            Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
+                        }
+                        if (Settings.ShowLatency)
```

Comment:
This uses the toggle under OnlyAreaName and applies it in both cases, dropping the submenu altogether and adding the toggles for fps and latency in the main xph menu would make more sense I think now that you mention it